Main.java

```diff
@@ -9,7 +10,40 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    // Scanner for user input
+    Scanner input = new Scanner(System.in);
+    
+    // asks user for input
+    System.out.println("What is the size of the pizza (in cm):");
+
+    //user input
+    int number = input.nextInt();
+    
+    //prices
+    double ingredients = (number * 0.5);
+    double labor = (0.75);
+    double power = (0.99);
```

Comment:
These should really be constants. Remember, constants should always be named using UPPERCASE_LETTERS and have the word final in front of them.